json2html/jsonconv.py

```diff
@@ -35,30 +35,21 @@
 
 class Json2Html:
 
-    def convert(self, **kwargs):
+    def convert(self, json="", table_attributes='border="1"'):
         """
             Convert JSON to HTML Table format
         """
 
         # table attributes such as class, id, data-attr-*, etc.
         # eg: table_attributes = 'class = "table table-bordered sortable"'
-        table_attributes = None
-        if 'table_attributes' in kwargs:
-            table_attributes = kwargs['table_attributes']
-        else:
-            # by default HTML table border
-            table_attributes = 'border="1"'
         self.table_init_markup = "<table %s>" % table_attributes
 
-        json_input = None
-        if 'json' in kwargs and kwargs['json']:
-            unvalidated_input = kwargs['json']
-            if type(unvalidated_input) in text_types:
-                json_input = json.loads(unvalidated_input, object_pairs_hook=OrderedDict)
-            else:
-                json_input = unvalidated_input
+        if not json:
+            json_input = {}
+        elif type(json) in text_types:
+            json_input = json_parser.loads(json, object_pairs_hook=OrderedDict)
```

Comment:
This will throw error when first argument being passed is a non-empty string.

```
>>> json2html.convert('string')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "json2html/jsonconv.py", line 50, in convert
    json_input = json_parser.loads(json, object_pairs_hook=OrderedDict)
  File "/usr/local/Cellar/python/2.7.8_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 351, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File "/usr/local/Cellar/python/2.7.8_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/Cellar/python/2.7.8_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```